src/libs/NumberFormatUtils/index.native.js

```diff
@@ -8,8 +8,15 @@ import '@formatjs/intl-numberformat/polyfill';
 import '@formatjs/intl-numberformat/locale-data/en';
 import '@formatjs/intl-numberformat/locale-data/es';
 
```

Comment:
This file is completely the same for native and Web platforms. The only difference is that we need these Polyfills for Android.
Can we just have simple `NumberFormatUtils.js`? and move these imports to PlatformSetup/index.native.js